### ISSUE TITLE
Cannot open triggers in DB2 LUW databases

### ISSUE THREAD

**Issue by sbredenhof** [dbeaver/dbeaver#21250]
### Description

Whenever I try and open the folder for triggers on my DB2 database, it says: 
`Cannot invoke "org.jkiss.dbeaver.ext.db2.model.DB2Table.getDataSource()" because "table" is null`

The log says this:
`Cannot invoke "org.jkiss.dbeaver.ext.db2.model.DB2Table.getDataSource()" because "table" is null
java.lang.NullPointerException: Cannot invoke "org.jkiss.dbeaver.ext.db2.model.DB2Table.getDataSource()" because "table" is null
	at org.jkiss.dbeaver.ext.db2.model.DB2Trigger.<init>(DB2Trigger.java:91)
	at org.jkiss.dbeaver.ext.db2.model.cache.DB2TriggerCache.fetchObject(DB2TriggerCache.java:64)
	at org.jkiss.dbeaver.ext.db2.model.cache.DB2TriggerCache.fetchObject(DB2TriggerCache.java:1)
	at org.jkiss.dbeaver.model.impl.jdbc.cache.JDBCObjectCache.loadObjects(JDBCObjectCache.java:119)
	at org.jkiss.dbeaver.model.impl.jdbc.cache.JDBCObjectCache.getAllObjects(JDBCObjectCache.java:74)
	at org.jkiss.dbeaver.ext.db2.model.DB2Schema.getTriggers(DB2Schema.java:364)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.extractPropertyValue(DBNDatabaseNode.java:966)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode$PropertyValueReader.run(DBNDatabaseNode.java:1038)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode$PropertyValueReader.run(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.loadTreeItems(DBNDatabaseNode.java:595)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.loadChildren(DBNDatabaseNode.java:464)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:227)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.navigator.DBNUtils.getNodeChildrenFiltered(DBNUtils.java:92)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:49)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:1)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:88)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:72)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)`

I have 2 db2 LUW databases and I cannot open the triggers on either of them.

I didn't think it was an issue with the database, but I confirmed by opening it in Datagrip and it worked fine.

The error happens in the folder directly inside the schema, as well as the one inside the table in the folder structure.

### DBeaver Version

Community 23.1.5.202308201919

### Operating System

Windows 10 Pro 22H2

### Database and driver

Database: DB2 LUW v11.5
Driver: com.ibm.db2:jcc:RELEASE [11.5.7.0]

### Steps to reproduce

1. Add db2 luw database that has triggers
2. try open triggers folder

### Additional context

No extensions being used.

**Comment by LonwoLonwo** [comment]
Hello @sbredenhof 

Did you try to reconnect?
For now, for some reason, the trigger table or schema can not be found by name.

> as well as the one inside the table in the folder structure.

Please provide a table DDL and a trigger DDL of these objects to recreate the issue.

Also, could you please check this query:

`SELECT * FROM SYSCAT.TRIGGERS WHERE TRIGSCHEMA = <your_schema_name> ORDER BY TRIGNAME WITH UR`

Do all names in the "TABNAME" match your tables list in the navigator tree?

**Comment by sbredenhof** [comment]
Sorry for the late response. I've been trying to replicate it myself on a different computer and I haven't been able to yet so it's probably an issue with my other computer or the database I was connecting to. 

I have tried reconnecting and it did not help. I also executed the query you gave and they did all match.

I actually can't provide the DDL for the database since its not mine and it is also very big. If there is any other information I can provide that might help you solve this, just ask. Otherwise feel free to close the case and I can play around with it some more since I can't help with that.
